utils/validate_icons_and_logos.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const { readPackFile, findMainPackConfigFiles } = require('./helpers');
+
+/**
+ * Method to validate icons and logos exist if supplied.
+ * @returns {string[]} error messages for errors encountered
+ */
+const validateIconAndLogo = () => {
+  const errorMessages = [];
+
+  for (const configPath of findMainPackConfigFiles()) {
```

Comment:
If you pass in the return value of `findMainPackConfigFiles`, it would probably make it easier to test. You won't need to stub it out since you can just pass in an array of file paths.

Reply:
do you mean like:
```js
const configPaths = findMainPackConfigFiles();

for(const configPath of configPaths){
}
```

i think i still need to mock the output of the method. this was also the way i originally had it and just had a thought to combine the lines 🙈